iTechArt.Api/Program.cs

```diff
@@ -37,7 +37,7 @@
 builder.Services.AddScoped<IXmlParse, ParseXml>();
 builder.Services.AddScoped<ICsvParse, ParseCsv>();
 builder.Services.AddScoped<IMedStaffParser, MedStaffParser>();
-builder.Services.AddScoped<IGroceryParsers, GroceryParsers>();
+builder.Services.AddScoped<IGroceryParser, GroceryParsers>();
```

Comment:
You have changed interface name from IGroceryParserS to IGroceryParser, but you didn`t change class name.
Why?